src/app/components/configure-app/configure-app.component.ts

```diff
@@ -113,21 +112,29 @@ export class ConfigureAppComponent implements OnInit {
   representativeListMatch = '';
 
   serverAPI = null;
+  serverAPIUpdated = null;
   serverWS = null;
   serverAuth = null;
   minimumReceive = null;
 
   showServerValues = () => this.selectedServer && this.selectedServer !== 'random';
   showServerConfigs = () => this.selectedServer && this.selectedServer === 'custom';
 
+  nodeBlockCount = 'N/A';
+  nodeUnchecked = 'N/A';
+  nodeCemented = 'N/A';
+  nodeUncemented = 'N/A';
+  nodeVendor = 'N/A';
+  nodeNetwork = 'N/A';
```

Comment:
Why not start with `null` (no value) and update it later with the correct value?
The block counts can be formatted as a number then as well.

Reply:
Because if the backend server does not support the version call for example I want it to display N/A because that's what it is, not available. If setting it to null, it will either be blank or we have to encode it to N/A in the HTML instead. I figured this method would be easier.

Reply:
As stated at Discord I would move that to the template anyway, I don't see why this is easier.
You just write `{{ nodeBlockCount || 'N/A' }}` and it's done. Additionally you get strict types (like `Number` for the block counts).